README.md

```diff
@@ -137,6 +138,18 @@ For more detailed information and advanced usage, please refer to the following:
 
 For updating versions, please refer to the [**Release Notes**](docs/release-notes/release-notes-0-to-14.md).
 
+## Build the Project
+
+You can build the project using Maven:
+
+```shell
+mvn clean install -DskipTests -DskipFormatting
```

Comment:
```suggestion
mvn clean install -DskipTests
```

I think we can remove the formatting skip.
* I don't think it has a computational footprint.
* It's not relying on external resources, it's stable.